Dockerfile

```diff
@@ -22,6 +22,9 @@ RUN pip install --progress-bar=off -r requirements/default.txt && \
 
 COPY . /app
 
+# File must exist otherwise volume is mounted as directory.
+RUN touch /app/config.toml
```

Comment:
I don't really understand this. Isn't `/app` being mounted as directory what we want? Were you able to get it to work mounting just a file?

Reply:
You're right, I got confused. It's when the source does not exist that a directory gets created.